pagespeed/apache/apache_rewrite_driver_factory.cc

```diff
@@ -227,4 +236,18 @@ ApacheConfig* ApacheRewriteDriverFactory::NewRewriteOptionsForQuery() {
   return new ApacheConfig("query", thread_system());
 }
 
+GoogleString ApacheRewriteDriverFactory::MasterPidFilename() {
+  if (ap_pid_fname == NULL) {
+    LOG(DFATAL) << "ap_pid_fname should not be null";
+    return "";
+  }
+  const char* fname = ap_server_root_relative(pool_, ap_pid_fname);
```

Comment:
Hmm, this is potentially slightly leaky if this gets called a lot, which it probably isn't.